src/utils.ts

```diff
@@ -25,6 +36,14 @@ export function parseUrl(url: string): Url {
   return urlObject;
 }
 
+export function posixify(pathname: string): string {
+  return pathname.replace(/\\/g, '/');
+}
+
+export function usePathSep(pathname: string): string {
```

Comment:
I went with `normalizePathSeparators` -if that's too vague I can change to your suggestion.